gluten-ut/spark34/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala

```diff
@@ -82,9 +82,11 @@ class VeloxTestSettings extends BackendTestSettings {
     // NEW SUITE: disable as it expects exception which doesn't happen when offloaded to gluten
     .exclude(
       "INCONSISTENT_BEHAVIOR_CROSS_VERSION: compatibility with Spark 2.4/3.2 in reading/writing dates")
-    .exclude("FAILED_EXECUTE_UDF: execute user defined function")
+    // JDBC test
     .exclude("UNRECOGNIZED_SQL_TYPE: unrecognized SQL type -100")
+    // gluten throws different exception
```

Comment:
Can you help investigate why the different exception occurs? And does it expect ?

Reply:
In spark, spark exception is thrown but in gluten in subquery transformer illegal state exception is thrown but the message is same so I have replicated the test in gluten side suite.